### PR TITLE
More docs about auto-initialization

### PR DESCRIPTION
**Pull request by nia-e**
Because we just can't have nice things. Follow-up on discussion in #79

### CODE REVIEW

**Comment by nia-e** [comment]
@rafaelcaricio is this good?

File: lvgl/Cargo.toml [hunk 1]
```diff
@@ -105,8 +110,3 @@ required-features = ["alloc", "embedded_graphics"]
 name = "sdl"
 path = "../examples/sdl.rs"
 required-features = ["alloc", "drivers"]
-
-[[example]]
-name = "memtest"
-path = "../examples/memtest.rs"
-required-features = ["lvgl_alloc"]
```
Comment (nia-e) [inline lvgl/Cargo.toml:112]: This was an oopsie, forgot to delete earlier when I was testing stuff. Shouldn't have ever made it to master 😓

File: lvgl/Cargo.toml [hunk 2]
```diff
@@ -60,7 +60,12 @@ nightly = []
 # segault. Only enable this if you're really, seriously, absolutely certain
 # that you need it. Unless you're doing something particularly exotic, this is
 # *not* needed, nor would it make anything measurably faster.
-unsafe_no_ctor = []
+#
+# However, if you're running baremetal or on a platform that ctor doesn't
+# support (https://github.com/mmastrac/rust-ctor/blob/master/ctor/src/lib.rs),
+# this feature should be enabled. Remember to call lvgl::init() in your main
+# function before constructing or using anything LVGL-related.
+unsafe_no_autoinit = []
```
[the remark this answers is not in the document]
Comment (nia-e) [reply lvgl/Cargo.toml:68]: It doesn't, no. I'll go do that

**Comment by nia-e** [comment]
Updated README. Anything else?

**Review by rafaelcaricio — APPROVED** [review]

**Pull request merged**